frontend/src/views/admin/UsersView.vue

```diff
@@ -57,25 +61,36 @@ const roleOptions = computed(() => {
     return roles.toSpliced(0, 1);
 });
 
+/**
+ * FillCreators fills a dictionary that links a role to the function that creates an instance of that role in the backend
+ */
 const fillCreators = (): void => {
     for (let i = 1; i < roles.length; i++) {
         const role: Role = roles[i];
         creators.value[role] = createFunctions.value[i - 1];
     }
 };
-
+/**
+ * FillDestroyers fills a dictionary that links a role to the function that destroys an instance of that role in the backend
+ */
 const fillDestroyers = (): void => {
```

Comment:
why do this in 2 seperate functions instead of in 1 (fillCreators, fillDestroyers)